### PR TITLE
Adds link to full certificate chain for Rsyslog

### PR DESCRIPTION
**Pull request by le-he-hoo**
### What does this PR do?

Adds a link to the full certificate chain to use for TLS for Rsyslog.

### Motivation

We have seen a number of tickets where customers are unable to use the certificate we link to the in the docs.  Using the full certificate chain has helped those customers move forward, and so, I wanted to add it to the docs.

### CODE REVIEW

File: content/logs/log_collection/rsyslog.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ description: "Configure Rsyslog to gather logs from your host, containers & serv
     sudo apt-get install rsyslog-gnutls
     ```
 
-    Download the public key for TLS encryption of logs from [this link][1]. Save it to `/etc/ssl/certs/intake.logs.datadoghq.com.crt`
+    Download the public key for TLS encryption of logs from [this link][1]. Save it to `/etc/ssl/certs/intake.logs.datadoghq.com.crt`.  
+    
+    **Note**: On some systems, the full certificate chain may be required.  If so, please download using [this link][2].
```
Comment (l0k0ms) [inline content/logs/log_collection/rsyslog.md]: @le-he-hoo is there a pattern on systems incriminated, like all linux/windows system?

Comment (le-he-hoo) [reply content/logs/log_collection/rsyslog.md]: I _believe_ it has mostly been Ubuntu with versions of Rsyslog >= 8.x

Comment (l0k0ms) [reply content/logs/log_collection/rsyslog.md]: 🤔 well let's be fuzzy on this one then...

**Comment by jeremy-lq** [comment]
@l0k0ms let's get these moved out of personal gists and into the repo/org.

**Comment by l0k0ms** [comment]
@jeremy-lq done, PR is here: https://github.com/DataDog/documentation/pull/2573

**Pull request merged**